File: src/prog_langs/java/for.java
class For{
    public static void main(String[] args) {
        // textにstring型で配列を格納
        String[] text = {"test","hoge","piyo","huga"};

        // 回すときの条件は0から始まり2未満まで、1ずつ増やす
        for (int i = 0; i < 2; i++){
          System.out.println(text[i]);
        }
    }
}

File: src/prog_langs/go/for.go
package main

import "fmt"

func main() {
  arr := []string{"hoge", "piyo", "huga", "test"}

  fmt.Println(arr)
  // 0番目から1ずつ増やして4回まわす
  for i := 0; i < 4; i++{
      fmt.Println(arr[i])
  }

  // 実行結果
  // [hoge piyo huga test]
  // hoge
  // piyo
  // huga
  // test
}